packages/algob/test/lib/ssc.ts

```diff
@@ -56,10 +56,9 @@ describe("Convert integer to big endian", () => {
   });
 
   it("should throw error if value is not uint64", () => {
-    let errMsg = "Invalid uint64 -5";
-    assert.throws(() => uint64ToBigEndian(MIN_UINT64 - 5n), errMsg);
+    const errMsg = "Input is not a 64-bit unsigned integer";
 
-    errMsg = "Invalid uint64 18446744073709551620";
+    assert.throws(() => uint64ToBigEndian(MIN_UINT64 - 5n), errMsg);
```

Comment:
does sdk functions don't throw error ? maybe we can add a check before. 
this seems more clear `Invalid uint64 18446744073709551620` (in terms of debugging also it is good)

Reply:
yes sdk function throws error. ok, adding check